.vsts-ci.yml

```diff
@@ -71,5 +90,12 @@ jobs:
       versionSpec: 4.7.0
       checkLatest: false
 
-  - bash: msbuild /r /p:Configuration=Release /p:AddSyntheticProjectReferencesForSolutionDependencies=false ./src/Uno.SourceGenerator.Linux.sln
+  - bash: msbuild /r /p:Configuration=Release ./src/Uno.SourceGenerator.Linux.sln # /bl:$(build.artifactstagingdirectory)/macos-build.binlog
```

Comment:
commented code?

Reply:
It's often required to add it back for troubleshooting, it's easier to leave it here :)